.circleci/wait-for-ssh.py

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env python
```

Comment:
let's rename this script and move it under a folder:`.circleci/build-workbase-win/startup-wait-for-ssh.py`.